lib/MySQL_HostGroups_Manager.cpp

```diff
@@ -15,13 +15,6 @@
 
 #include <mutex>
 
-#define USE_MYSRVC_ARRAY
-
-#ifdef USE_MYSRVC_ARRAY
-static unsigned long long array_mysrvc_total = 0;
-static unsigned long long array_mysrvc_cands = 0;
```

Comment:
`array_mysrvc_total` and `array_mysrvc_cands` are used if `TEST_AURORA` is defined.
Every time they are used, they should be wrapped by `#ifdef TEST_AURORA`